setup.py

```diff
@@ -18,7 +18,7 @@
         "pyOpenSSL~=18.0.0",
         "PyYAML>=5.1",
         "intervaltree~=3.0.2",
-        "importlib_metadata; python_version<'3.8'",
+        "importlib_metadata",
```

Comment:
Is this right?
isn't `importlib_metadata` already in py3.8? (so this should not be needed?)

https://docs.python.org/3/library/importlib.metadata.html#module-importlib.metadata

Reply:
i see that now, let me try building without `importlib_metadata` included in setup.py

Reply:
i see, the importlib was made part of python3.8, but the syntax will have to change where it is used in this repo. Additionally, the features of importlib_metadata that were included in python3.8 are equivalent to importlib_metadata version 1, whereas this was already tested with importlib_metadata version 7.0.1. I'd recommend keeping the backport library in here as a result.